docs/shared/features/run-tasks.md

```diff
@@ -1,12 +1,91 @@
 # Run Tasks
 
-Monorepos can have hundreds or even thousands of projects, so being able to run actions against all (or some) of them is a key feature of a tool like Nx.
+In a monorepo setup, you don't just run tasks for a single project; you might have hundreds to manage. To help with this, Nx provides a powerful task runner that allows you to:
 
-## Types of Tasks
+- **speed up task execution** with caching
+- easily **run multiple targets** for multiple projects **in parallel**
+- define **task pipelines** to run tasks in the correct order
+- only run tasks for **projects affected by a given change**
```

Comment:
I would put caching at the bottom of this list since it isn't the main topic of this page.

```suggestion
- easily **run multiple targets** for multiple projects **in parallel**
- define **task pipelines** to run tasks in the correct order
- only run tasks for **projects affected by a given change**
- **speed up task execution** with [caching](/features/cache-task-results)
```